test/swift_client_test.rb

```diff
@@ -341,6 +341,18 @@ def test_get_objects
     assert_equal objects, @swift_client.get_objects("container-1", :limit => 2, :marker => "object-2").parsed_response
   end
 
+  def test_get_objects_no_json
+    objects = [
+     "object-2",
+     "object-3",
+    ]
+    objects_as_string = objects.map { |o| "#{o}\n" }.join
```

Comment:
```suggestion
    response = ["object-2", "object-3"].join("\n")
```

imo, no need for 4 lines with not much semantics.

Reply:
The server does emit a trailing newline. But good point, I just unrolled the loop 👍